rocket/rocket.py

```diff
@@ -79,7 +82,7 @@ def launch(
             project_name = os.path.abspath(project_location).split("/")[-1]
             dbfs_path = f"{dbfs_path}/{project_name}"
 
-        self._build_and_deploy(watch, project_location, dbfs_path)
+        self._build_and_deploy(watch=watch, project_location=project_location, dbfs_path=dbfs_path, glob_path=glob_path)
         if watch:
             watcher = FileWatcher(
```

Comment:
I think you would need to remove the file limiter in the filewatcher to make it work. Currently we only look at python files

Reply:
AFAIK there is no filter applied in FileWatcher. Can you double check?

Reply:
I found it ... as it is implemented right now the watch feature would not work for `non-python` files. :(